### ISSUE TITLE
Can this run more than once?

### ISSUE THREAD

**Issue by wicadmin** [qb60/hikvision-configure#1]
Would running this more than once on the same camera mess up anything?

**Comment by qb60** [comment]
No, wouldn't. You can run it as many times as you need. But if new and old password aren't the same the script won't work the second time because camera will already has new password. 
And the script won't do activation the second time if it was already done on this camera.

**Comment by wicadmin** [comment]
Great. I tried to remove the OSD info from the live view and the web ui seems broken for that:
![image](https://user-images.githubusercontent.com/4563854/45489724-121e7080-b733-11e8-81f9-f4fa8351b786.png)

I also tried to so some manual PUT curls based on the 583 page API doc and got some "device error". However, changing the appropriate attribute to 'false' in your program and re-running it work!

Thanks.
